Python/optimizer.c

```diff
@@ -1721,4 +1722,14 @@ _Py_Executors_InvalidateAll(PyInterpreterState *interp, int is_invalidation)
     }
 }
 
+#else
+
+void
+_Py_Executors_InvalidateDependency(PyInterpreterState *interp, void *obj, int is_invalidation)
```

Comment:
I'm not super keen on having a dummy version here -- and if we do this we should probably make a few more dummy versions and remove the occasional `#if _Py_TIER2` sprinkled around elsewhere.

Reply:
Would you prefer this in optimizer.h
```C
#else
#define _Py_Executors_InvalidateDependency(A, B, C) ((void)0)
#endif _Py_TIER2
```
and leave optimizer.c alone?

Reply:
That's better. Though if this is what Brandt asks about maybe we should split it and put that bugfix in beta1 even if the f_locals bugfix goes to beta2.